### PR TITLE
Added script to run KafkaSource tests with KEDA

### LINKED ISSUE #3067: Test KafkaSource with KEDA
**Problem**

We're not testing the KafkaSource integration with KEDA, we need to have automated tests running in CI (in a separate job) for the KEDA integration.

### PR DESCRIPTION
**Pull request by Cali0707**
Part of #3067 

## Proposed Changes

- Add a config map to enable autoscaling in tests
- Add a script which enables autoscaling in tests then runs the KafkaSource tests

### CODE REVIEW

File: test/keda-reconciler-tests.sh
Comment (matzew) [inline test/keda-reconciler-tests.sh:1]: Do we introduce new "infra" hooks for that, so there is a new test target?

Comment (Cali0707) [reply test/keda-reconciler-tests.sh:1]: Yeah I'll add that today

Comment (Cali0707) [reply test/keda-reconciler-tests.sh:1]: https://github.com/knative/infra/pull/188

**Review by pierDipi — APPROVED** [review]
/lgtm
/approve

**Comment by Cali0707** [comment]
@pierDipi any ideas why the channel integration tests might keep failing? I don't think I've changed anything related to them in this PR...

**Pull request merged**